curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/iterations-more-patterns.md

```diff
@@ -3,6 +3,10 @@ id: 5e7b9f070b6c005b0e76f05f
 title: 'Iterations: More Patterns'
 challengeType: 11
 videoId: 9Wtqo6vha1M
+bilibiliIds:
+  aid: 334491369
+  bvid: BV1tw411R7Mm
+  cid: 376530765
```

Comment:
These are the ids for https://www.freecodecamp.org/learn/scientific-computing-with-python/python-for-everybody/iterations-loop-idioms do you have the correct ones?

Reply:
[Here](https://www.bilibili.com/video/BV1tw411R7Mm?spm_id_from=333.999.0.0) is the video. The ids are correct. I don't what's wrong with it.

`<iframe src="//player.bilibili.com/player.html?aid=334491369&bvid=BV1tw411R7Mm&cid=376530765&page=1" scrolling="no" border="0" frameborder="no" framespacing="0" allowfullscreen="true"> </iframe>`